src/transformers/pipelines/text_generation.py

```diff
@@ -37,10 +37,10 @@ def __init__(self, messages: Dict):
 class TextGenerationPipeline(Pipeline):
     """
     Language generation pipeline using any `ModelWithLMHead`. This pipeline predicts the words that will follow a
-    specified text prompt. It can also accept one or more chats. Each chat takes the form of a list of dicts,
-    where each dict contains "role" and "content" keys.
+    specified text prompt. When the underlying model is a conversational model, it can also accept one or more chats.
+    Each chat takes the form of a list of dicts, where each dict contains "role" and "content" keys.
```

Comment:
Maybe specify that when doing so, the model will automatically behave in chat mode?